test/cypress/integration/group-payments.spec.js

```diff
@@ -251,7 +251,7 @@ describe('Payments', () => {
   })
 
   it('user3 pledges $100 and sees who they are pledging to - $50 to user1 (Greg)', () => {
-    cy.giSwitchUser(`user3-${userId}`, { bypassUI: true })
+    cy.giSwitchUser(`user3-${userId}`)
```

Comment:
Note: By default giSwitchUser always uses bypassUI, so there's no need to pass the argument.